mailchimp_auth/views.py

```diff
@@ -223,13 +230,25 @@ def post(self, *args, **kwargs):
                     'to access this tool.'
                 )
                 form.errors['email'] = [error_message.format(email=form.cleaned_data['email'])]
+                
+                if os.getenv("SENTRY_DSN"):
+                    err_msg = "Following user not found during login: {}. They most likely entered an unsubscribed email.".format(form.cleaned_data['email'])
+                    sentry_sdk.capture_message(err_msg, "warning")
+                    send_error_email(err_msg, "login", self.request)
+
```

Comment:
Do you think we need this instance to be logged? This is essentially just when a user tries to log in with an unrecognized email, whether as a typo or misremembering.

Reply:
Let's do more logging now, and we can toggle it off later.